docs/public/specifications/tests/Address-TP/address01.md

```diff
@@ -10,38 +21,94 @@ name servers must have addresses in the routable public addressing space.
-and [Multicast reserved
-addresses](https://www.iana.org/assignments/multicast-addresses/multicast-addresses.xml).
+for special uses. These ranges can be categorized into three types:
+
+* [Special purpose IPv4 addresses]
+* [Special purpose IPv6 addresses]
+
+### Scope
+
+IP addresses from authoritative name servers are matched against IANAs list of public routable address ranges.
 
 ### Inputs
 
-The domain name to be tested.
+* The domain name to be tested.
+
+### Summary
 
-### Ordered description of steps to be taken to execute the test case
+Message Tag                       | Level    | Arguments | Message ID for message tag
+:-------------------------------- |:---------|:----------|:--------------------------
+A01_ADDR_GLOBALLY_REACHABLE       | INFO     | ns_list   | IP addresses in "{ns_list}" listed as globally reachable.
+A01_NO_GLOBALLY_REACHABLE_ADDR    | CRITICAL | ns_list   | None of the IP addresses in "{ns_list}" listed as globally reachable.
+A01_ADDR_NOT_GLOBALLY_REACHABLE   | ERROR    | ns        | IP adress for "{ns}" not listed as globally reachable.
+A01_DOCUMENTATION_ADDR            | ERROR    | ns        | IP adress for "{ns}" part of address range for documentation purposes.
+A01_LOCAL_USE_ADDR                | ERROR    | ns        | IP adress for "{ns}" part of address range intended for local use on network or service provider level..
+
+
+The value in the Level column is the default severity level of the message. The
+severity level can be changed in the [Zonemaster-Engine profile]. Also see the
+[Severity Level Definitions] document.
+
+The argument names in the Arguments column lists the arguments used in the
+message. The argument names are defined in the [Argument list].
+
+### Test procedure 
 
 1. Obtain the IP addresses of each name server of the domain from the parent using
-   [Method4](../Methods.md) and child using [Method5](../Methods.md)
+   [Method4] and [Method5]. 
 
-2. If any IP address (IPv4 or IPv6) falls within any of the blocks in the below
-   mentioned IANA links, the test case fails:
-   * <https://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xhtml>
-   * <https://www.iana.org/assignments/iana-ipv6-special-registry/iana-ipv6-special-registry.xhtml>
+2. For each name server in *Name Server IP* do:
+   1. Match the IP the address against the IP ranges specified in [Special purpose IPv4 addresses] and [Special purpose IPv6 addresses]
+      1. If if falls within any of the address ranges reserved for *Documentation*, then output *[A01_DOCUMENTATION_ADDR]* with name and IP address of the server.
+      2. Else, if it falls within an address range belonging to any of the following categories:  
+         - *Private-Use*
+         - *Loopback*
+         - *Loopback Address*
+         - *Link Local*
+         - *Link-Local Unicast*
+         - *Unique-Local*
+         - *Shared Address Space*
```

Comment:
Maybe `*Private-Use* (IPv4)`, `*Unique-Local* (IPv6` etc. In that way it will be easier to read.